aviary/variable_info/variable_meta_data.py

```diff
@@ -2192,6 +2192,18 @@
     option=True
 )
 
+add_meta_data(
+    Aircraft.Engine.RPM_DESIGN,
+    meta_data=_MetaData,
+    historical_name={"GASP": None,
```

Comment:
possible similar variable:
INPROP.XNMAX = maximum engine speed, rpm

Reply:
I added this change.